config/katello.migrations/210415175000-enable-katello-agent.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman_proxy_content'] && answers['foreman_proxy_content']['enable_katello_agent'].nil?
+  answers['foreman_proxy_content']['enable_katello_agent'] = true
+end
```

Comment:
since this is the katello scenario migration, should this be `answers['katello']` throughout instead?

or does it work because the foreman.hiera for katello scenario includes the alias `katello::params::enable_katello_agent: "%{alias('foreman_proxy_content::enable_katello_agent')}"` ?

Reply:
> or does it work because the foreman.hiera for katello scenario includes the alias `katello::params::enable_katello_agent: "%{alias('foreman_proxy_content::enable_katello_agent')}"` ?

Correct. Everything is driven off of the user facing paramter `--foreman-proxy-content-enable-katello-agent`